### DISCUSSION TITLE
My account flagged

### DISCUSSION THREAD

**Discussion by emrahyilm4z** [community/community#147732] [New to GitHub]
Hello, everyone.
My account has been closed for over a month. I can't get any kind of response even though I send a ticket. What should I do? Does anyone have any idea?

**Comment by emrahyilm4z** [comment]
Hi guys. I first submitted a ticket on November 14, 2024, and the first response from Github support was on January 12, 2025. The reason why my account was flagged is that I have two different accounts. When I deleted my additional account, the flag was removed.

**Reply by fayaznasrati** [reply]
Is it possible to have more than one GitHub account with a single email address, 
OR
 can having multiple accounts with different email addresses cause an account to be flagged? 
 For example, I currently have a GitHub account under the username “fayaznasrati” using fayaz.nasraty@gmail.com,
 and another under “fayaz10” using fayaz.nasrat10@gmail.com. Could this lead to a flagged account?
  
@chtholine @emrahyilm4z, assist please .

**Comment by chtholine** [comment]
Thank you for reaching out to our community and providing an update on your support experience. I understand the delay in response must have been frustrating. It’s good to hear that the issue was resolved after deleting the additional account.
If you have any further concerns or questions, feel free to reach out!
Since the question is answered, could you mark this comment as an answer? This would help me get a Galaxy Brain achievement 😃

**Comment by ghostinhershell** [comment]
Hi @emrahyilm4z,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
